packages/json-rpc-engine/src/JsonRpcEngine.ts

```diff
@@ -200,9 +202,13 @@ export class JsonRpcEngine extends SafeEventEmitter {
     requests: (JsonRpcRequest<Params> | JsonRpcNotification<Params>)[],
   ): Promise<JsonRpcResponse<Result>[]>;
 
-  // TODO: Replace `any` with type
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  handle(req: unknown, callback?: any) {
+  handle(
+    req:
+      | (JsonRpcRequest | JsonRpcNotification)[]
+      | JsonRpcRequest
+      | JsonRpcNotification,
+    callback?: (error: unknown, response: never) => void,
```

Comment:
`never` is used to widen the implementation signature enough to be compatible with all of the overload signatures.